packages/common/src/types.ts

```diff
@@ -319,11 +348,25 @@ export interface IHubContent extends IHubResource, IItem {
   data?: {
     [propName: string]: any;
   };
+  // service types: Feature Service, Map Service
+  /** service information (currentVersion, capabilities, maxRecordCount etc) */
+  server?: Partial<IFeatureServiceDefinition>;
```

Comment:
Should this be `service`? vs `server`?

Reply:
the Hub API returns `server`, so we rollin' w/ that.